doc/snippets/Microsoft.Data.SqlClient/SqlException.xml

```diff
@@ -1,390 +1,889 @@
+        </para>
+        <para>
+          The <see cref="T:Microsoft.Data.SqlClient.SqlConnection" /> remains open when the severity level is 19 or less. When the severity level is 20 or greater, the server ordinarily closes the <see cref="T:Microsoft.Data.SqlClient.SqlConnection" />. However, the user can reopen the connection and continue. In both cases, a <see cref="T:Microsoft.Data.SqlClient.SqlException" /> is generated by the method executing the command.
+        </para>
+        <para>
+          For information about the warning and informational messages sent by SQL Server, see <see href="/sql/relational-databases/errors-events/database-engine-events-and-errors">Database Engine Events and Errors</see>. The <see cref="T:Microsoft.Data.SqlClient.SqlException" /> class maps to SQL Server severity.
+        </para>
+        <para>
+          The following is general information on handling exceptions. Your code should catch exceptions to prevent the application from crashing and to allow displaying a relevant error message to the user.  You can use database transactions to ensure that the data is consistent regardless of what happens in the client application (including a crash). Features like <see cref="T:System.Transactions.TransactionScope" /> or the <see cref="M:Microsoft.Data.SqlClient.SqlConnection.BeginTransaction" /> ensure consistent data regardless of exceptions raised by a provider. Transactions can fail, so catch failures and retry the transaction.
+        </para>
+        <para>
+          Note that beginning with .NET Framework 4.5, <see cref="T:Microsoft.Data.SqlClient.SqlException" /> can return an inner <see cref="T:System.ComponentModel.Win32Exception" />.
+        </para>
+        <para>
+          The exception class of a .NET Framework data provider reports provider-specific errors. For example System.Data.Odbc has <see cref="T:System.Data.Odbc.OdbcException" />, System.Data.OleDb has <see cref="T:System.Data.OleDb.OleDbException" />, and Microsoft.Data.SqlClient has <b>SqlException</b>. For the best level of error detail, catch these exceptions and use the members of these exception classes to get details of the error.
+        </para>
+        <para>
+          Bad input can cause a .NET Framework data provider type to raise an exception such as System.ArgumentException or System.IndexOutOfRangeException. Calling a method at the wrong time can raise System.InvalidOperationException.
+        </para>
+        <para>
+          So, in general, write an exception handler that catches any provider specific exceptions as well as exceptions from the common language runtime. These can be layered as follows:
+        </para>
+        <code language="c#">
+          try
+          {
```

Comment:
remove these?

Reply:
Honestly these examples seem totally pointless to me because it's basically "here's how to use a try/catch and catch SqlException separately". But I'm not really trying to change the contents here.

Was your comment just about the extra space?

Reply:
It was about the contents of the example. I don't think they're adding any value. I'm fine with leaving it if it's out of scope for your effort.